### PR TITLE
Updating disk and memory with queries and changing variable names

### PR DESCRIPTION
**Pull request by emataei**

### CODE REVIEW

File: armtemplates/alerts/azuredeploy_Disk.json [hunk 1]
```diff
@@ -95,9 +95,9 @@
       ],
       "properties": {
         "Type": "Alert",
-        "description": "CPU Usage is above 90%",
+        "description": "Disk Usage is above 90%",
         "severity": "informational",
```
Comment (aburek) [inline armtemplates/alerts/azuredeploy_Disk.json]: Should this be "Remaining disk space is under 100MB"?

File: armtemplates/alerts/azuredeploy_Disk.parameters.json [hunk 2]
```diff
@@ -12,16 +12,16 @@
             "value": "YourActionGroupName"
         },
         "alertName": {
-            "value": "CPU Alert"
+            "value": "Disk Alert"
         },
         "searchName": {
-            "value": "CPUPercentTest"
+            "value": "DiskPercentTest"
```
Comment (jason-lachance) [inline armtemplates/alerts/azuredeploy_Disk.parameters.json]: It's not a percent -- it's under 100 MB. Needs an accurate name, please.

File: armtemplates/alerts/azuredeploy_Disk.json (hunk 1 above)
Comment (aburek) [inline armtemplates/alerts/azuredeploy_Disk.json]: "severe"

File: armtemplates/alerts/azuredeploy_Memory.json [hunk 3]
```diff
@@ -95,9 +96,9 @@
       ],
       "properties": {
         "Type": "Alert",
-        "description": "CPU Usage is above 90%",
+        "description": "Memory Usage is above 90%",
         "severity": "informational",
```
Comment (aburek) [inline armtemplates/alerts/azuredeploy_Memory.json]: "severe"

File: armtemplates/alerts/azuredeploy_Disk.parameters.json [hunk 4]
```diff
@@ -12,16 +12,16 @@
...
         },
         "scheduleName": {
             "value": "Schedule"
         },
         "customEmailSubject": {
-            "value": "CPU alert"
+            "value": "Disk alert"
```
Comment (jason-lachance) [inline armtemplates/alerts/azuredeploy_Disk.parameters.json]: If I were to receive this email, I might prefer something like "ALERT: Disk space low!" which would both catch my attention and make it easy to search for all of the "ALERT" emails

File: armtemplates/alerts/azuredeploy_Memory.parameters.json [hunk 5]
```diff
@@ -12,16 +12,16 @@
...
         "searchName": {
-            "value": "CPUPercentTest"
+            "value": "MemoryPercentTest"
         },
         "scheduleName": {
             "value": "Schedule"
         },
         "customEmailSubject": {
-            "value": "CPU alert"
+            "value": "Memory alert"
```
Comment (jason-lachance) [inline armtemplates/alerts/azuredeploy_Memory.parameters.json]: Same email subject comment as in ```azuredeploy_Disk.parameters.json```

File: armtemplates/alerts/azuredeploy_Memory.json [hunk 6]
```diff
@@ -65,8 +65,9 @@
       "properties": {
         "mode": "Incremental",
         "Category": "AlertIbiza",
-        "DisplayName": "CPU Usage Alert",
-        "Query": "Perf\n| where TimeGenerated > ago(1m) \n| where CounterName == \"% Processor Time\" and InstanceName == \"_Total\"\n| summarize AggregatedValue = avg(CounterValue) by Computer\n| where AggregatedValue > 90",
+        "DisplayName": "Memory Usage Alert",
+        "Query": "Perf\n| where TimeGenerated > ago(1m)\n| where CounterName == \"% Processor Time\" and InstanceName == \"_Total\"\n| summarize AggregatedValue = avg(CounterValue) by Computer\n| where AggregatedValue > 90",
```
Comment (aburek) [inline armtemplates/alerts/azuredeploy_Memory.json]: Should this one still be > 90%? Or will we use a different metric?

**Review by jason-lachance — CHANGES_REQUESTED** [review]
Couple of minor things...

File: armtemplates/alerts/azuredeploy_Memory.json (hunk 6 above)
Comment (jason-lachance) [reply armtemplates/alerts/azuredeploy_Memory.json]: Pretty sure this one is actually OK

**Review by aburek — APPROVED** [review]

**Pull request merged**